cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-storage.adoc

```diff
@@ -0,0 +1,119 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-storage"]
+= Tutorial: Storage
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-storage
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-04-30
+
+== Persistent volumes
+ROSA supports storing persistent volumes with either link:https://aws.amazon.com/ebs/[Amazon Web Services (AWS) Elastic Block Store (EBS)] or link:https://aws.amazon.com/efs/[AWS Elastic File System (EFS)]. For more information about persistent volume storage, see xref:../../storage/understanding-persistent-storage.adoc#persistent-volumes_understanding-persistent-storage[Understanding persistent storage].
+
+== Using persistent volumes
+Use the following procedures to create a file, store it on a persistent volume in your cluster, and confirm that it still exits after pod failure and re-creation.
+
+=== Viewing a persistent volume claim
```

Comment:
Before "Viewing a persistent volume claim", I think it's better to introduce how to create the PVC and how the PVC is consumed by the application/pod, or add the step on how the OSToy app and its PVC come.

Reply:
I added some additional resources for users that want to learn more before beginning the procedure. Thanks!